config/locales/client.en.yml

```
@@ -6336,9 +6336,11 @@ en:
           install: "Install"
           themes:
             title: "Themes"
+            description: "Site-wide themes that change the overall look and feel of your site for all users"
```

```suggestion
            description: "Site-wide themes that change the overall look and feel of your site for all users."
```

Small change to match the acceptance criteria.